CHANGELOG.md

```diff
@@ -2,8 +2,18 @@
 
 ## [unreleased]
 
+### Protocol Changes
+**No Changes**
+
+### Non-protocol Changes
```

Comment:
Do we want to mention the state sync changes?

Reply:
Good point, somehow just forgot it. Added it now